api/auth/jwtMiddleware.test.js

```diff
@@ -17,8 +17,6 @@ tap.test('jwtMiddleware', async t => {
   const user = { name: 'dude' };
   const payload = {
     uid: '1234',
-    groups: ['eAPD State Coordinator'],
-    affiliations: ['MO'],
     hasLoggedIn: true
```

Comment:
you can get rid of hasLoggedIn too, we won't be getting that from Okta any more either